src/Core/Settings/GlobalSettings.cs

```diff
@@ -742,4 +743,9 @@ public class WebPushSettings : IWebPushSettings
     {
         public string VapidPublicKey { get; set; }
     }
+
+    public class Fido2Settings
+    {
+        public HashSet<string> Origins { get; set; }
```

Comment:
ℹ️ I am assuming the configuration readers in .NET can work with this type but an array or list of strings is more typical.